bot.py

```diff
@@ -325,15 +364,35 @@ async def _appeal_accept(self, ctx, *, reason):
             except:
                 return
 
-    @commands.has_any_role(config.modRole)
-    @_appeal.command(name='deny')
-    async def _appeal_deny(self, ctx, next_attempt, *, reason):
+    @cog_ext.cog_subcommand(
+        base='appeal',
+        name='deny',
+        guild_ids=guildList,
+        base_description='Make a decision to accept or deny a ban appeal',
+        description='Deny a user\'s ban appeal',
+        base_permissions={config.guild: [create_permission(config.modRole, SlashCommandPermissionType.ROLE, True)]},
```

Comment:
`{config.guild: [create_permission(config.modRole, SlashCommandPermissionType.ROLE, True)]}` should be made into a static variable and passed instead.